### DISCUSSION TITLE
Continuous - erroring out

### DISCUSSION THREAD

**Discussion by mc-cabe** [juftin/camply#65] [Q&A] [answered]
I set up a yml file to search a few campsites. once It runs through all the sites, it seems to error out.

I have continue to true and polling int to 5

2022-02-02 09:33:30,941 [   ERROR]: No campgrounds found to search
2022-02-02 09:33:30,942 [  CAMPLY]: Exiting camply 👋
..
    raise SearchError(error_message)
camply.search.base_search.SearchError: No campgrounds found to search

**Comment by juftin** [comment, marked as answer]
Can you share the exact commands you're paassing to `camply` and the full output? It looks like you're providing invalid campground / recreation area / campsite IDs and it's unable to resolve it to actual campgrounds to search.

**Reply by mc-cabe** [reply]
oh wow recreation_area != campgrounds.. im an idiot.

**Reply by juftin** [reply]
Glad you got it working @mastermccabe!

**Reply by mc-cabe** [reply]
yeah this is awesome; i'd love to be able to contribute in the future
